advocacy_docs/partner_docs/EsriArcGISPro/04-ConfiguringEsriArcGISPro.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Configuring Esri ArcGIS Pro'
+description: 'Walkthrough on configuring Esri ArcGIS Pro'
+---
+
+Implementing Esri ArcGIS Pro with EDB Postgres Advanced Server requires the following components:
+
+- A running EDB Postgres Advanced Server instance
+- PostGIS version 3.2 or above, installed on EDB Postgres Advanced Server
+- Your GIS data
+
```

Comment:
What about 
A running Esri ArcGIS Pro instance

Reply:
That sounds good. Finalizes the list. Adding it now.